pkgs/top-level/python-packages.nix

```diff
@@ -22378,6 +22378,29 @@ in {
     };
   };
 
+  pyslurm = buildPythonPackage rec {
```

Comment:
No. We do this for all new python packages and for most updated ones, due the size of this file.